cheatsheets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -0,0 +1,387 @@
+# Introduction
+
+The objective of the cheat sheet is to provide advices regarding the protection against [Server Side Request Forgery](https://www.acunetix.com/blog/articles/server-side-request-forgery-vulnerability/) attack.
+
+**S**erver **S**ide **R**equest **F**orgery will be named **SSRF** in the rest of the cheat sheet.
+
+This cheat sheet will focus on the defense point of view and will not explains how to perform this attack. This [talk](../assets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet_Orange_Tsai_Talk.pdf) from the security researcher [Orange Tsai](https://twitter.com/orange_8361) as well as this [document](../assets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet_SSRF_Bible.pdf) provides deep advices about how to perform this kind of attack.
+
+# Context
+
+SSRF is a way to force an application to make a malicious network request. It can happen when a user can control the URL to an external resource like: 
```

Comment:
OK i have a little bit adapted to: 

*SSRF is an attack vector that abuses an application to interact with the internal/external network or the machine itself. One of the enablers for this vector is the mishandling of URLs, as showcased in the following examples*

Because SSRF is not always intended for *internal* network even if it's the most common usage.

Reply:
Taken in account;